managedkafka/examples/src/main/java/examples/DeleteCluster.java

```diff
@@ -38,13 +45,38 @@ public static void main(String[] args) throws Exception {
 
   public static void deleteCluster(String projectId, String region, String clusterId)
       throws Exception {
-    try (ManagedKafkaClient managedKafkaClient = ManagedKafkaClient.create()) {
+
+    // Create the settings to configure the timeout for polling operations
```

Comment:
Same here, we should only set .setTotalTimeoutDuration(Duration.ofHours(1L))

Reply:
Fixed!